test/TensorFlow/integration.swift

```diff
@@ -417,10 +417,26 @@ public func testResourceAndVariants() {
 // CHECK:  [[VALUES:%.*]] = graph_op "Const"() {dtype: $Float, value$tensor: [$Float: (f32 0x3F800000 /* 1 */), (f32 0x40000000 /* 2 */),
 // CHECK:  [[DATASET:%.*]] = graph_op "TensorDataSet,i"([[VALUES]] : $TensorHandle<Float>
 // CHECK:  [[ITERATOR:%.*]] = graph_op "Iterator"()
-// CHECK:  graph_op "MakeIterator,i,i"([[DATASET]] : $VariantHandle, [[ITERATOR]] : $ResourceHandle) {{.*}} 
+// CHECK:  graph_op "MakeIterator,i,i"([[DATASET]] : $VariantHandle, [[ITERATOR]] : $ResourceHandle) {{.*}}
 // CHECK-LABEL: ----
 
 
+public func testStringHandle() {
+  let str: StringTensorHandle = #tfop(
+    "Const", dtype: String.self, value$tensor: "foo"
```

Comment:
Please also add a few more tests to create string tensors with higher dimensions. You can do that by passing an array literal of strings to the `value$tensor:` attribute.

Reply:
Added a 1d tensor test